src/reducers/ProgressTrackerReducer.jsx

```diff
@@ -49,6 +49,20 @@ const ProgressTrackerReducer = (state = initialState, action) => {
       const ahseTotal = sum(ahseCourses);
       const sciTotal = sum(sciCourses);
 
+      // converts progress into a string that can be used by
+      // the progress bar in the ProgressTracker component
+      const mathSciPercNum = Math.round((mathTotal + sciTotal) * 3.33);
```

Comment:
This (hard-coded factors, toString() with concat) is super gross. Can this be a function or something? Or can we perhaps do the string conversion in the component's curly braces?